src/modules/mc_pos_control/mc_pos_control_main.cpp

```diff
@@ -159,7 +159,7 @@ class MulticopterPositionControl : public ModuleBase<MulticopterPositionControl>
 		(ParamInt<px4::params::MPC_POS_MODE>) MPC_POS_MODE,
 		(ParamInt<px4::params::MPC_AUTO_MODE>) MPC_AUTO_MODE,
 		(ParamInt<px4::params::MPC_ALT_MODE>) MPC_ALT_MODE,
-		(ParamFloat<px4::params::MPC_IDLE_TKO>) MPC_IDLE_TKO, /**< time constant for smooth takeoff ramp */
+		(ParamFloat<px4::params::MPC_TKO_DELAY>) MPC_TKO_DELAY, /**< time between arming and takeoff */
```

Comment:
I don't think this name is better. I agree with @bresch, `MPC_SPOOL_UP_TIME' or something similar would be better